src/utilities/effects.ts

```diff
@@ -800,6 +853,139 @@ const effects = {
     }
   },
 
+  async createExternalEventType(eventType: ExternalEventTypeInsertInput, user: User | null) {
+    try {
+      creatingExternalEventType.set(true);
+      createExternalEventTypeError.set(null);
+      if (eventType) {
+        const { createExternalEventType: created } = await reqHasura<any>(
```

Comment:
Can we specify the return type in all these queries that have `reqHasura<any>`?

Reply:
Most of these have been implemented, however the queries for `getExternalEventTypes`, `getExternalEventTypesBySource`, and `getExternalEvents` have been particularly problematic and have not yet been updated.

Reply:
Resolved.